app/views/staff/proposals/_form.html.haml

```diff
@@ -10,11 +10,20 @@
       %section.inline-block
         = f.label :slides_url
         = f.text_field :slides_url, class: "form-control"
-      %p
-      = f.select :review_tags,
-        options_for_select(event.review_tags, proposal.object.review_tags),
-        {}, {class: 'multiselect review-tags', multiple: true}
-
+  .row
+    %fieldset.col-md-6
+      %section
+        %p &nbsp;
+        = f.label :review_tags, label: "Review Tags (#{event.review_tags.length})"
```

Comment:
Yeah, we want any aspect of the UI that serves tags to go away such as labels, column headers, section, etc that are only there for displaying tags if event.public_tags? returns false.